keyboards/redscarf_iiplus/verd/keymaps/alittlepeace/keymap.c

```diff
@@ -0,0 +1,86 @@
+ */
+#include QMK_KEYBOARD_H
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+    KC_P00 = SAFE_RANGE,
+    CTRLMA,
+
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    [0] = LAYOUT_60_numpad_split0(
+        KC_GESC, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_BSPC, TG(1), KC_PSLS, KC_PAST, KC_PMNS, 
+        KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, KC_BSLS, KC_INS, KC_HOME, KC_PGUP, 
+        KC_CAPS, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_ENT, KC_DEL, KC_END, KC_PGDN, KC_PPLS, 
+        KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_RSFT, 
+        CTRLMA, KC_UP, LCTL(KC_TAB), KC_LCTL, KC_LGUI, KC_LALT, KC_SPC, KC_RALT, KC_RGUI, MO(2), KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT, KC_PENT
+    ),
+
+    [1] = LAYOUT_60_numpad_split0(
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_7,    KC_8,    KC_9,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, KC_4,    KC_5,    KC_6,    _______,
+        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, KC_1,    KC_2,    KC_3,
+        _______, _______, _______,                            _______,                            _______, _______, _______, _______, KC_0,    KC_P00,  KC_DOT,  _______
+    ),
+    
+    [2] = LAYOUT_60_numpad_split0(
+        _______, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _____,  _____,  _____,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _____,  _____,  _____,    _______,
+        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _____,  _____,  _____,
+        _______, _______, _______,                            _______,                            _______, _______, _______, _______, _____,  _____,  _____,  _______
+    ),
+};
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case KC_P00:
+            if (record->event.pressed) {
+                SEND_STRING("00");
+                 }
+            break;
+        case CTRLMA:
+            if (record->event.pressed) {
+		tap_code16(C(S(KC_TAB)));

```

Comment:
You can actually just put `C(S(KC_TAB))` directly into the keymap. 

If you want it shorter, you could instead use:
```
#define CTRLMA C(S(KC_TAB))
```
And then just use `CTRLMA` in your layout.